langchain/callbacks/base.py

```diff
@@ -229,6 +248,16 @@ async def on_llm_error(
     ) -> None:
         """Run when LLM errors."""
 
+    async def on_chat_model_error(
```

Comment:
why no `on_chat_model_end`?  Is it because chat models also returns an `LLMResult`?

Reply:
Ya - meant to remove the error one too. I can just add all three and check for 'chat' in the handle event like recommended